### DISCUSSION TITLE
Paho mqtt

### DISCUSSION THREAD

**Discussion by cabildocl** [flet-dev/flet#4463] [Packages] [answered]
https://pypi.org/project/paho-mqtt/

**Comment by FeodorFitsner** [comment, marked as answer]
It's a pure Python package - should be already supported on mobile platforms.

Two notes:
- paho-mqtt 2.x requires you to pick the callback API version, e.g. `client = mqtt.Client(mqtt.CallbackAPIVersion.VERSION2)`.
- It's an MQTT *client* — it connects to a remote broker over the network (there's no embedded broker on the phone); TLS works via the stdlib `ssl` module.
